layouts/default.vue

```diff
@@ -392,9 +307,8 @@ export default {
       console.log('Start NCHAN from mount')
       this.startNCHAN(me.id)
 
-      // Get notifications and chats and poll regularly for new ones.  Would be nice if this was event driven instead but requires server work.
+      // Get chats and poll regularly for new ones.  Would be nice if this was event driven instead but requires server work.
       this.fetchLatestChats()
```

Comment:
We've lost a call to updateNotifications here. Is that intentional?

Reply:
Which one?  The updateNotifications call which was in mounted in default.vue has been moved into the mounted of notificationoptions.vue.

Although I have noticed that I can also move the fetchchats in to the relevant component....